tools/run-app.js

```diff
@@ -751,7 +744,7 @@ _.extend(AppRunner.prototype, {
 
         // Notify the server that new client assets have been added to the
         // build.
-        appProcess.proc.kill('SIGUSR2');
+        appProcess.proc.send({ refresh: 'client' });
```

Comment:
The docs say "Emits an 'error' event if the message cannot be sent, for example because the child process has already exited".  (presumably on the proc object?)  now, we do have an error handler on the proc object but it logs "Couldn't spawn process" which is wrong in this case.  see the docs: http://nodejs.org/api/child_process.html#child_process_event_error